policy/README.md

```diff
@@ -203,7 +203,9 @@ Here's how it works in practice:
 
 
 
-Agencies that wish to use Geography-Driven Events do so by requiring a new `event_geographies` field in status events. This field acts in lieu of location or telemtry data. When an Agency is using Geography-Driven Events, Providers must emit a new `changed_geographies` status event whenever a vehicle in a trip enters or leaves a Geography managed by a Policy.
+Agencies that wish to use Geography-Driven Events do so by requiring a new `event_geographies` field in status events. When an Agency is using Geography-Driven Events, Providers must emit a new `changed_geographies` status event whenever a vehicle in a trip enters or leaves a Geography managed by a Policy.
+
+During the Beta period for this feature, location and telemtry data remain required fields. This allows Aggencies to test Geometry-Driven Events, measuring its accuracy and efficacy against regulatory systems based on precise location data. After the beta period, if Geometry-Driven Events is deemed by OMF to be accurate and effective, the specification will evolve to allow cities to use Geography-Driven Events in lieu of location or telemtry data.
```

Comment:
I wrote 'Geometry' on purpose here just to maintain the long running joke. Changed it back to 'Geography' in the subsequent commit.

Reply:
I for one thought this was much funnier than modifying my Git history